src/screens/Waivers/WaiverManager.jsx

```diff
@@ -33,27 +113,102 @@ function authWrapper(Component) {
 
 const WaiverManager = () => {
   const [waivers, setWaivers] = useState({});
+  const [adultExists, setAdult] = useState(true);
+  const [minorExists, setMinor] = useState(true);
+  const [adultWaiver, setAdultWaiver] = useState(null);
+  const [minorWaiver, setMinorWaiver] = useState(null);
+  const [user, setUser] = useState(null);
+
+  // const getAndSetWaivers = async () => {
+  //   const res = await getWaivers();
+  //   setWaivers(res.data);
+  // };
+
+  if (!user) {
+    const { data: session } = useSession();
+    setUser(session.user);
+  }
 
-  const getAndSetWaivers = async () => {
-    const res = await getWaivers();
-    setWaivers(res.data);
+  // const onRefresh = () => {
+  //   loadWaivers();
+  // };
+
+  const loadWaivers = async () => {
+    const adult = await getWaivers("adult", user.organizationId);
+    setAdultWaiver(adult);
+
+    const minor = await getWaivers("adult", user.organizationId);
+    setMinorWaiver(minor);
   };
 
-  useEffect(() => {
-    getAndSetWaivers();
-  }, []);
+  const editWaiver = async () => {
+    await router.push(`${router.pathname}/edit`);
+  };
+
+  // useEffect(() => {
+  //   getAndSetWaivers();
+  // }, []);
 
   return (
     <WaiversContainer>
       <h2>Manage Waivers</h2>
-      <Waiver
+
+      <WaiverContainer>
+        {adultExists ? (
+          <>
+            <WaiverUploadContainer>
+              <WaiverHeader>Adult Waiver</WaiverHeader>
+              <WaiverUploadForm>
+                <SubmitButton type="editPage" onClick={editWaiver}>
+                  Edit Waiver
```

Comment:
See my comment in slack. We don't need these containers/buttons anymore, just two tabs with text boxes for the respective waiver and a save button